pkg/kepval/approval.go

```diff
@@ -100,15 +100,10 @@ func ValidatePRR(kep *api.Proposal, h *api.PRRHandler, prrDir string) error {
 func isPRRRequired(kep *api.Proposal) (required, missingMilestone, missingStage bool, err error) {
 	logrus.Debug("checking if PRR is required")
 
-	required = true
+	required = kep.Status == api.ImplementableStatus
 	missingMilestone = kep.IsMissingMilestone()
 	missingStage = kep.IsMissingStage()
 
-	if kep.Status != "implementable" {
-		required = false
-		return required, missingMilestone, missingStage, nil
-	}
-
```

Comment:
I'd prefer to keep this and move to L104 to do an early return when possible:

```go
if !required {
    return required, missingMilestone, missingStage, nil
}

Reply:
I want us to start dropping warnings about this data missing, because it will be necessary if we want to change logic to check for PRR if status is "implementable"